Cargo.lock

```diff
@@ -5241,9 +5237,9 @@ dependencies = [
 
 [[package]]
 name = "stringdex"
-version = "0.0.1-alpha10"
+version = "0.0.1-alpha9"
```

Comment:
bad rebase?

Reply:
I think lexical sorting is leading to alpha10 being treated as the version between alpha1 and alpha2, so `cargo update` pulls in the 'newest' version, alpha9
<img width="585" height="178" alt="image" src="https://github.com/user-attachments/assets/1df41f79-56b1-4238-92ff-45c1b9fb4048" />
<img width="155" height="145" alt="image" src="https://github.com/user-attachments/assets/d1456bdd-f79e-46fe-9137-96d47d8a08dc" />

Reply:
Ouch, good point.

The `=` version is fine for now. I'll switch to 0.0.2 next time I push an update.